.env

```
@@ -4,15 +4,37 @@ MENDER_SERVER_REPOSITORY=mendersoftware
 MENDER_SERVER_ENTERPRISE_REPOSITORY=mender-server-enterprise
 MENDER_SERVER_TAG=main
 
+MENDER_REGISTRY=registry.mender.io
+MENDER_DOCKER_REGISTRY=docker.io
```

To follow the convention above (server) and below (gateway), these should be:
```suggestion
MENDER_CLIENT_REGISTRY=docker.io
MENDER_CLIENT_ENTERPRISE_REGISTRY=registry.mender.io
```

I apologize I didn't comment on this on the first pass of this PR.